### ISSUE TITLE
Can NUVOTON NCT6683D Super I / O chips support

### ISSUE THREAD

**Issue by Hush-vv** [CloverHackyColor/FakeSMC3_with_plugins#13]
Can NUVOTON NCT6683D Super I / O chips support？
<img width="1061" alt="73E9EC1B919659D71A293D5850632772" src="https://user-images.githubusercontent.com/61744850/121805317-5f7df280-cc7d-11eb-8018-4767d54491e7.png">

**Comment by SergeySlice** [comment]
Yes, but you should choose  configuration of IO ports. It can't be universal and I don't know your configuration.
